src/pages/GetInvolvedPage.tsx

```diff
@@ -1,5 +1,36 @@
+import { Map, MapGeoJsonSource, MapContainer } from "@/atoms/MapAtoms";
+import { Theme } from "@/theme/Theme";
+import  BPDA from "@/data/BPDA";
+import { Marker } from "react-map-gl";
+
+const BPDADevelopmentLayer = () => (
+  <MapGeoJsonSource
+    data={BPDA}
+    id="BPDA"
+    type="circle"
+    color={Theme.colors.blue}
+  />
+);
+
 export default function GetInvolvedPage() {
   return (
-    <h1>Get Involved</h1>
-  )
+    <MapContainer>
+      <Map mapStyle={Theme.map.dark}>
+        <>
+          <BPDADevelopmentLayer />
```

Comment:
Since you're using Marker component straight from react-map-gl, I don't actually think you'll need this component. MapGeoJsonSource was originally really for boundary-like features (i.e looking at lots of coordinates to make a boundary) as opposed to a collection of points where each coordinate pair is a defined location